exercises/wordy/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: "java"
+apply plugin: "eclipse"
+apply plugin: "idea"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  testCompile "junit:junit:4.12"
+}
+test {
+  testLogging {
+    exceptionFormat = 'full'
+    events = ["passed", "failed", "skipped"]
```

Comment:
I added `showStandardStreams = true` here while I was testing my implementation; could be something we consider as standard in the future to help folks?